tests/test_core.py

```diff
@@ -7476,12 +7485,15 @@ def test_async_ccall_promise(self, exit_runtime):
   ccall('stringf', 'string', ['string'], ['first\n'], { async: true })
     .then(function(val) {
       console.log(val);
-      ccall('floatf', 'number', null, null, { async: true }).then(console.log);
+      ccall('floatf', 'number', null, null, { async: true }).then(function(arg) {
+        console.log(arg);
+        maybeExit();
```

Comment:
Is this still needed? I think it's part of the stuff you said you'd revert.

Reply:
Reverted.

Reply:
I still see `maybeExit()` here, weird. Is the github UI confusing me maybe?

Reply:
Oops, actually reverted this now.